glide.yaml

```diff
@@ -72,3 +72,14 @@ import:
   subpackages:
   - compute/metadata
   - internal
+- package: k8s.io/kubernetes
+  version: e4a1ca7105fa9a3075fbf13a3286f8dddd59a658
```

Comment:
Okay, so I had to do the following, since it seems like the repo for the google cloud libraries may have moved and, not only did they move, but the import URL changed. This will keep the existing import URL. (Glide can't figure it out due to the import URL changing i think)
```
diff --git a/glide.yaml b/glide.yaml
index de8f578..2f93117 100644
--- a/glide.yaml
+++ b/glide.yaml
@@ -30,7 +30,7 @@ import:
   - activation
   - daemon
 - package: github.com/coreos/pkg
-  version: v2 
+  version: v2
   subpackages:
   - flagutil
 - package: github.com/golang/glog
@@ -68,7 +68,9 @@ import:
 - package: google.golang.org/appengine
   version: 6a436539be38c296a8075a871cc536686b458371
 - package: google.golang.org/cloud
-  version: 872c736f496c2ba12786bedbb8325576bbdb33cf
+  repo: https://github.com/GoogleCloudPlatform/gcloud-golang.git
+  vcs: git
+  version: eb47ba841d53d93506cfbfbc03927daf9cc48f88
   subpackages:
   - compute/metadata
   - internal
```